client/src/app/shared/models/agenda/item.ts

```diff
@@ -61,6 +61,13 @@ export class Item extends ProjectableBaseModel {
         return this.speakers.filter(speaker => speaker.state === SpeakerState.WAITING).length;
     }
 
+    /**
+     * Gets the string representation of the item type
+     */
+    public get verboseType(): string {
+        return this.type ? itemVisibilityChoices[this.type].name : '';
```

Comment:
use `.find(choice => choice.key === this.type)`. The index is not euqal to the key.